ai_serving/fasttext_clf/test.txt

```diff
@@ -0,0 +1,5 @@
+what can you do
```

Comment:
Make a separate directory and move all the data files in there for better organization.